.gitignore

```diff
@@ -1 +1 @@
-.idea
```

Comment:
good to keep editors

Reply:
imho this should be part of global gitignore not related to the repo

Reply:
I must say I agree but good to prevent errors since we already have a diff for it